packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec

```diff
@@ -1,25 +1,18 @@
 # template: foreman_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
-
 %global gem_name foreman_expire_hosts
 %global plugin_name expire_hosts
 %global foreman_min_version 3.0.0
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 8.1.0
+Name: rubygem-%{gem_name}
+Version: 8.2.0
 Release: 1%{?foremandist}%{?dist}
 Summary: Foreman plugin for limiting host lifetime
-Group: Applications/Systems
 License: GPLv3
 URL: https://github.com/theforeman/foreman_expire_hosts
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
-Source1: %{gem_name}.cron.d
```

Comment:
Is this intentional? If so, I'd expect this file to be removed from the packaging repo too

Reply:
Oh thanks for noticing, this was not intentional. I missed that the recreation of the spec from the template deleted that part.

Reply:
I changed the commited and added `Source1` back into the spec.